### DISCUSSION TITLE
Welcome to CodingSpace Discussions!

### DISCUSSION THREAD

**Discussion by rishipurwar1** [rishipurwar1/FrontendPro#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by irffanasiff** [comment]
Hii can you plese make a new issue with hacktoberfest tag where i will submit my design and animation that i have done for roadmaps page

**Reply by rishipurwar1** [reply]
will do that soon
